src/App.tsx

```diff
@@ -2,84 +2,128 @@ import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+type Props = {};
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  goods: string[],
+  chosenElem: string,
+  clearBtn: boolean,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<Props, State> {
+  state = {
+    goods: [
+      'Dumplings',
+      'Carrot',
+      'Eggs',
+      'Ice cream',
+      'Apple',
+      'Bread',
+      'Fish',
+      'Honey',
+      'Jam',
+      'Garlic',
+    ],
+    chosenElem: 'Jam',
+    clearBtn: true,
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  chosenGood = (good: string) => {
+    this.setState({ chosenElem: good });
+    this.setState({ clearBtn: true });
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  clear = () => {
+    this.setState({ chosenElem: 'No' });
```

Comment:
```suggestion
    this.setState({ chosenElem: '' });
```

better to set just an empty string, and check if this value empty